build-utils/language-c/src/main/java/org/apache/plc4x/language/c/CLanguageTemplateHelper.java

```diff
@@ -178,33 +178,44 @@ public boolean requiresPointerAccess(ComplexTypeDefinition typeDefinition, Strin
      */
     @Override
     public String getLanguageTypeNameForTypeReference(TypeReference typeReference) {
+        System.out.println("Type Reference " + typeReference.getClass());
```

Comment:
Small leftover to be cleaned up.

Reply:
(the fix is contained in my mega review below 😉 )